sdk/storage/storage-blob/README.md

```diff
@@ -108,6 +100,23 @@ For example, you can create following CORS settings for debugging. But please cu
 - Exposed headers: \*
 - Maximum age (seconds): 86400
 
+## Key concepts
+
+Blob storage is designed for:
+
+- Serving images or documents directly to a browser.
+- Storing files for distributed access.
+- Streaming video and audio.
+- Writing to log files.
+- Storing data for backup and restore, disaster recovery, and archiving.
+- Storing data for analysis by an on-premises or Azure-hosted service.
```

Comment:
on-premise

Reply:
No, should have the `s` according to [wikipedia](https://en.wikipedia.org/wiki/On-premises_software).